### DISCUSSION TITLE
folder name case sensitivity

### DISCUSSION THREAD

**Discussion by StarlitScarf** [Raytwo/Cobalt#35] [Suggestions]
it's a pretty simple suggestion, but I noticed that the current release folder name for the game ID is all lowercase making it so it isn't drag and drop with Linux due to it's case sensitivity. It's easily solved by moving the contents to a properly capitalized folder, but it would be nice to get this changed.

**Comment by Raytwo** [comment]
Got it, just made the change for it so it should be reflected on the next release. 👌
